medicines/api/src/main.rs

```diff
@@ -38,9 +39,17 @@ async fn healthz() -> impl actix_web::Responder {
     "OK"
 }
 
+fn cors_middleware() -> actix_cors::CorsFactory {
+    Cors::new()
+        .allowed_methods(vec!["POST"])
+        .allowed_headers(vec![http::header::AUTHORIZATION, http::header::ACCEPT])
+        .allowed_header(http::header::CONTENT_TYPE)
+        .max_age(3600)
```

Comment:
Do you not need to specify the allowed origins as well?

Reply:
@sgrowe We deliberately decided to not set allowed origins as this is a public API. One of the acceptance criteria is that it can be used by third party developers.

I'll look into whether we can combine `allowed_headers` and `allowed_header`.